src/OrbitGl/TimeGraphTest.cpp

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <gtest/gtest.h>
+
+#include "CaptureViewElementTester.h"
+#include "TimeGraph.h"
+#include "TrackTestData.h"
+#include "Viewport.h"
+
+namespace orbit_gl {
+
+class UnitTestTimeGraph : public testing::Test {
+ public:
+  explicit UnitTestTimeGraph() {
+    capture_data_ = TrackTestData::GenerateTestCaptureData();
+    viewport_ = std::make_unique<Viewport>(100, 200);
+    time_graph_ = std::make_unique<TimeGraph>(nullptr, nullptr, viewport_.get(),
+                                              capture_data_.get(), nullptr);
+  }
+
+  // TODO(b/230441102): Temporal solution while we implement mouse moved methods for
+  // CaptureViewElements.
+  void MouseMoved(int x, int y, bool left, bool right, bool middle) {
+    if (!(left || right || middle)) {
+      time_graph_->ProcessSliderMouseMoveEvents(x, y);
+    }
+  }
+
+  void SimulatePreRender() {
+    CaptureViewElementTester tester;
```

Comment:
I would not recreate `CaptureViewElementTester` every time, but keep it as a class member. It doesn't matter for those tests, but it's meant to be a class that persists until the end of the test so you can use the contained Batcher, Viewport and Layout throughout the test. If we start adding rendering tests in this test class, this may give the wrong impression of how to use `CaptureViewElementTester`.